pkg/build/node-shim.js

```diff
@@ -15,7 +19,11 @@ async function main() {
   }
 
   try {
+    {{if .IsCodeDefinedTask}}
+    let ret = await {{.EntrypointFunc}}._airplane.baseFunc(JSON.parse(process.argv[2]));
```

Comment:
Minor: can we do double-underscores to match the convention of internal fields elsewhere? (e.g. `__airplaneType`).

```suggestion
    let ret = await {{.EntrypointFunc}}.__airplane.baseFunc(JSON.parse(process.argv[2]));
```